DMX.Core.Api.Tests.Acceptance/DMX.Core.Api.Tests.Acceptance.csproj

```diff
@@ -9,12 +9,12 @@
 
   <ItemGroup>
     <PackageReference Include="DeepCloner" Version="0.10.4" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="6.0.6" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.0-preview.5.22303.8" />
```

Comment:
Didn't realize we were doing preview versions of nuget packages as well. Nice 😎

Reply:
@terrypalo bleeding-edge all the way :)